samples/subsys/mgmt/mcumgr/smp_svr/boards/mg100.conf

```diff
@@ -0,0 +1,4 @@
+# Enable QSPI (MX25R64) - Slot 1 in QSPI
+CONFIG_NORDIC_QSPI_NOR=y
+CONFIG_NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
+CONFIG_NORDIC_QSPI_NOR_STACK_WRITE_BUFFER_SIZE=4
```

Comment:
This needs to be done from a boards level perspective, not a sample perspective

Reply:
You do want this at the board level because if someone enables mcuboot and mcumgr for a random application, it would give a broken build due to not having QSPI enabled. Likewise with the modem, the modem should be selected automatically if networking is enabled, there should be no overlay in specific samples turning it on because anyone use a different sample will have a non working build.

Reply:
Yes, thanks for clarifying.  Pinnacle 100 DVK board needs to be fixed :)